composer.json

```diff
@@ -32,15 +32,14 @@
   },
   "require": {
     "php": "^7.0",
-    "symfony/console": "^3.2",
+    "symfony/console": "^3.3",
     "monolog/monolog": "^1.20"
   },
   "require-dev": {
-    "phpunit/phpunit": "^5.7",
+    "phpunit/phpunit": "^6.2",
     "browscap/browscap-php": "^3.1.0",
     "mikey179/vfsStream": "^1.6",
-    "squizlabs/php_codesniffer": "^2.8",
-    "friendsofphp/php-cs-fixer": "^1.11",
```

Comment:
I wonder if it's trying to iterate the JSON files too? In which case yes that makes sense :)